Maths/PalindromeNumber.java

```diff
@@ -0,0 +1,30 @@
+package Maths;
+
+public class PalindromeNumber {
+    public static void main(String[] args) {
+
+        assert (isPalindrome(12321));
+        assert (!isPalindrome(1234));
+        assert (isPalindrome(1));
```

Comment:
Hi @shellhub , please remove unnecessary parentheses. Like this,

```java
assert isPalindrome(12321);
assert !isPalindrome(1234);
assert isPalindrome(1);
```

Reply:
I agree! updated, Please review again!